NetworkManager.go

```diff
@@ -91,6 +106,27 @@ func (n *networkManager) ActivateWirelessConnection(c Connection, d Device, ap A
 	return nil
 }
 
+func (n *networkManager) AddAndActivateWirelessConnection(connection map[string]map[string]interface{}, d Device, ap AccessPoint) (ac ActiveConnection, err error) {
+	var opath1 dbus.ObjectPath
+	var opath2 dbus.ObjectPath
+	defer func() {
```

Comment:
Just curious, what can panic from this call?  `callError2` returns an error anyway.  Is this necessary?

Reply:
No, it is just some leftover code. Will remove...

Reply:
Thanks!